grc/core/generator/flow_graph.py.mako

```diff
@@ -28,6 +28,10 @@
 from packaging.version import Version as StrictVersion
 from PyQt5 import Qt
```

Comment:
I've seen this in some other places, but think it's caused some issues at least for me. I believe it's more idiomatic to import e.g. widgets from `QtWidgets`. I was always under the impression the `Qt` namespace was more for constants and enums like `Qt.AlignLeft` etc. When I run the test_examples test, I get loads of AttributeErrors about `QPushButton` not being a part of `PyQt5.Qt`